source/templates/_main.slim

```diff
@@ -65,12 +65,12 @@ aside[ng-controller="DefaultPopupController as defaultPopupCtrl"]
 				h2[ng-bind="defaultPopupCtrl.alertPopup.parameters.title"]
 				p[ng-repeat="detail in defaultPopupCtrl.alertPopup.parameters.details" ng-bind-html="detail"]
 			footer
-				button[ng-click="defaultPopupCtrl.alertPopup.isVisible = false" ng-bind="defaultPopupCtrl.alertPopup.parameters.okTitle"]
+				button.confirm[ng-click="defaultPopupCtrl.alertPopup.isVisible = false" ng-bind="defaultPopupCtrl.alertPopup.parameters.okTitle"]
 	popup[model="defaultPopupCtrl.confirmPopup"]
 		.popup-body#default-confirm-popup-body
 			article.vertical-scroll
 				h2[ng-bind="defaultPopupCtrl.confirmPopup.parameters.title"]
 				p[ng-repeat="detail in defaultPopupCtrl.confirmPopup.parameters.details" ng-bind-html="detail"]
 			footer
-				button.yes[ng-click="defaultPopupCtrl.confirmPopup.parameters.yesSelected()" ng-bind="defaultPopupCtrl.confirmPopup.parameters.yesTitle"]
-				button.no[ng-click="defaultPopupCtrl.confirmPopup.parameters.noSelected()" ng-bind="defaultPopupCtrl.confirmPopup.parameters.noTitle"]
+				button.yes.confirm[ng-click="defaultPopupCtrl.confirmPopup.parameters.yesSelected()" ng-bind="defaultPopupCtrl.confirmPopup.parameters.yesTitle"]
+				button.no.cancel[ng-click="defaultPopupCtrl.confirmPopup.parameters.noSelected()" ng-bind="defaultPopupCtrl.confirmPopup.parameters.noTitle"]
```

Comment:
@slapec93 if we need to touch elements in prod code can we use `data-e2e-tag="cancel-button"` or some similar tag? and then in selector: `.popup [data-e2e-tag="cancel-button"]`. So we do not interfere at all with styling